workloads/amd64/k3s-upgrade-plan.yaml

```diff
@@ -65,10 +99,7 @@ spec:
       - {key: node-role.kubernetes.io/etcd, operator: NotIn, values: ["true"]}
       - {key: node-role.kubernetes.io/control-plane, operator: NotIn, values: ["true"]}
   serviceAccountName: system-upgrade
-  prepare:
-    image: rancher/k3s-upgrade
-    args: ["prepare", "k3s-server-etcd"]
```

Comment:
same question - why did you remove the prepare? We want each of the roles to wait for the previous role's plan to finish before running. agent should wait for etcd, etcd should wait for cp.

It also looks like there are 4 plans here? I would probably have one yaml file for split-role clusters that had 3 plans (cp/etcd/agent) and another for etcd+cp clusters where you only have 2 plans (server/agent).